consent/models.py

```diff
@@ -239,3 +245,193 @@ class Meta(Consent.Meta):
+        '<li>course_start_date: Course start date. Available when type of consent is course.</li>'
+        '</ul>'
+    ))
+    POLICY_PARAGRAPH_HELP_TEXT = mark_safe_lazy(_(
+        'Fill in a text for policy paragraph at the bottom of page. The following variables may be available:<br />'
+        '<ul>'
+        '<li>enterprise_customer_name: A name of enterprise customer.</li>'
+        '<li>platform_name: Name of platform.</li>'
+        '<li>item: A string which is "course" or "program" depending on the type of consent.</li>'
+        '<li>course_title: Title of course. Available when type of consent is course.</li>'
+        '<li>course_start_date: Course start date. Available when type of consent is course.</li>'
+        '</ul>'
+    ))
+    SIDEBAR_PARAGRAPH_HELP_TEXT = mark_safe_lazy(_(
+        'Fill in a text for left sidebar paragraph. The following variables may be available:<br />'
+        '<ul>'
+        '<li>enterprise_customer_name: A name of enterprise customer.</li>'
+        '<li>platform_name: Name of platform.</li>'
+        '<li>item: A string which is "course" or "program" depending on the type of consent.</li>'
+        '<li>course_title: Title of course. Available when type of consent is course.</li>'
+        '<li>course_start_date: Course start date. Available when type of consent is course.</li>'
+        '</ul>'
+    ))
+    CONFIRMATION_MODAL_HELP_TEXT = mark_safe_lazy(_(
+        'Fill in a text for dialog which appears when user decline to provide consent. '
+        'The following variables may be available:<br />'
+        '<ul>'
+        '<li>enterprise_customer_name: A name of enterprise customer.</li>'
+        '<li>item: A string which is "course" or "program" depending on the type of consent.</li>'
+        '<li>course_title: Title of course. Available when type of consent is course.</li>'
+        '<li>course_start_date: Course start date. Available when type of consent is course.</li>'
+        '</ul>'
+    ))
+    NOTIFICATION_TITLE_HELP_TEXT = mark_safe_lazy(_(
+        'Fill in a text for title of the notification which appears on dashboard '
+        'when user decline to provide consent. '
+        'The following variables may be available:<br />'
+        '<ul>'
+        '<li>enterprise_customer_name: A name of enterprise customer.</li>'
+        '<li>course_title: Title of course. Available when type of consent is course.</li>'
+        '</ul>'
+    ))
+    NOTIFICATION_MESSAGE_HELP_TEXT = mark_safe_lazy(_(
+        'Fill in a text for message of the notification which appears on dashboard '
+        'when user decline to provide consent. '
+        'The following variables may be available:<br />'
+        '<ul>'
+        '<li>enterprise_customer_name: A name of enterprise customer.</li>'
+        '<li>course_title: Title of course. Available when type of consent is course.</li>'
+        '</ul>'
+    ))
```

Comment:
You should chunk up the `_data_sharing_policy.html` template into smaller template partials that match the chunks that you want to be customizable per EnterpriseCustomer. Make the fields in this model optional with no defaults. Then in each of the template partials you created by the chunking about, check the model to see if the chunk has an override.

Reply:
we opted to go with default values or override approach to 
1) give ability to skip any text/sections. i.e if an enterprise customer don't want to show policy dropdown they can set its values to empty.
2) defaults give users an idea of typical html for certain section

Reply:
@douglashall 
for the reasons @ziafazal pointed out, are you ok with going with the approach we have implemented?

Reply:
No. I really don't like defining this HTML as strings in python code. It should be defined in a template file where an engineer would expect HTML code to be defined. You can always define an override with an empty string value if that is what the desired override needs to be.

Reply:
@douglashall these HTML strings are going to be displayed only in django admin where staff can override those according to their needs. Should i override django admin template? I'm reluctant to do that for maintainability reasons.

Reply:
Yes, I have less of an issue with the help strings you are defining here. The default strings you have defined on the fields below are still a problem.